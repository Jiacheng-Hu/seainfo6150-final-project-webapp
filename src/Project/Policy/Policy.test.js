import React from "react";
import Policy from "./Policy";


describe("Policy tests", () => {
    it("renders correctly", () => {
      const { container } = render(<Policy/>);
      expect(container).toMatchSnapshot();
    });
  });