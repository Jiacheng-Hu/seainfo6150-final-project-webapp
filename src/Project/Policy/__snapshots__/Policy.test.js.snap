// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Policy tests renders correctly 1`] = `
<div>
  <div
    class="policy-main container"
  >
    <h1
      class="policy-title"
    >
      Privacy Information
    </h1>
    <h3>
      WELCOME
    </h3>
    <p>
      June 11,2018: (“University”, “Northeastern”, “we”, “us” or “our”) is a U.S.-based institution of higher learning offering undergraduate, graduate and continuing education programs, curriculum content and services, in a traditional campus setting and through our central websites (each a “Site”) Northeastern.edu, neu.edu, the my Northeastern portal, and our mobile applications (“Apps”) (collectively, the “Services”).
    </p>
    <h3>
      SCOPE
    </h3>
    <p>
      Personal Data. This Privacy Statement describes the type of personal data that we collect, how we use and share that data, and your choices concerning our data practices. Personal data (“Personal Data”) means any information about you from which you can be identified.
    </p>
    <p>
      By providing us with your Personal Data or using our Services, you agree to the data protection practices described in this Privacy Statement. If you do not agree to this Privacy Policy, please do not access or use the Services.
    </p>
    <p>
      For more information about this Privacy Statement, and your rights under applicable law, please contact us at 
      <a
        href="mailto:privacy@northeastern.edu"
      >
        privacy@northeastern.edu
      </a>
      .
    </p>
    <h3>
      WHAT INFORMATION DO WE COLLECT?
    </h3>
    <h4>
      Personal Data
    </h4>
    <p>
      We collect Personal Data that you voluntarily give us when you visit the Site, use the Apps or the Services to inquire about admission to the University, enroll in one of our programs, inquire about or obtain financial aid, apply or are hired to work at the University or participate in our alumni activities including fundraising. This data includes:
    </p>
    <ul>
      <li>
        Name, title, age and date of birth
      </li>
      <li>
        Social Security Number (SSN) as required by law for applicants for admission and financial aid, employees and for validation purposes, the last 4 digits are required for sponsored network account applicants
      </li>
      <li>
        Northeastern University (NUID) Number (as the student identification number and as an administrative system identifier for faculty and staff)
      </li>
      <li>
        Contact information including address, city, state, postal code, country of residence, email address, home, cellular and other contact telephone number(s)
      </li>
      <li>
        Campus address and telephone number
      </li>
      <li>
        Emergency contact information (names, phone numbers and email addresses)
      </li>
      <li>
        Academic credentials
      </li>
      <li>
        Academic, leisure or other interests
      </li>
      <li>
        Attendance at University events
      </li>
      <li>
        Records of communications between you and the university
      </li>
      <li>
        Other information pertinent to your specific interests and activities at Northeastern
      </li>
    </ul>
    <p>
      We also collect information you choose to provide to us when you complete any “free text” boxes in our forms (for example, support requests or blogs or forums).
    </p>
    <h4>
      Social Network or Third Party Plug-ins or Widgets
    </h4>
    <p>
      Our Services may include social network or third party plug-ins or widgets that may provide information to their social networks or third-parties about your interactions with our web pages, even if you do not click on or otherwise interact with the plug-in or widget. The information is transmitted from your browser and may include an identifier assigned by the social network or third party, information about your browser type, operating system, device type, IP address, and the URL of the web page where widget appears. Please read the privacy policies of these sites before you visit them or use their tools to learn what information they collect, use, and share. We are not responsible for the privacy policies or data practices of social network or third party sites or their party plug-ins or widgets.
    </p>
  </div>
</div>
`;
